arches/app/views/main.py

```diff
@@ -16,14 +16,17 @@
 along with this program. If not, see <http://www.gnu.org/licenses/>.
 """
 
-import re
-import urllib.request, urllib.error, urllib.parse
+import urllib.error
+import urllib.parse
+import urllib.request
 from urllib.parse import urlparse
+
+from django.conf import settings as project_settings
```

Comment:
Actually, I don't think we should be importing settings via the standard Django import until we address the behavior difference between dev and prod fleshed out in #11660.